save/src/lib.rs

```diff
@@ -139,6 +139,16 @@ impl Reader<'_> {
             .map_err(GetError::DecompressionFailed)?;
         Ok(Some(Character::decode(&*self.accum)?))
     }
+
+    /// Temporary function to load all voxel-related save data at once.
+    /// TODO: Replace this implementation with a streaming implementation
+    /// that does not require loading everything at once
+    pub fn get_all_voxel_node_ids(&mut self) -> Result<Vec<u128>, GetError> {
```

Comment:
nit:
```suggestion
    pub fn get_all_voxel_node_ids(&self) -> Result<Vec<u128>, GetError> {
```

Reply:
There are definitely legitimate use cases for using a `&mut self` in situations like this, which I hadn't thought about, so I had expected Clippy to catch things like this. Good catch!